src/strategies/DeltaNeutralLp.sol

```diff
@@ -84,25 +100,44 @@ contract DeltaNeutralLp is AccessStrategy {
         borrowPrice = uint256(price);
     }
 
-    /// @notice Get price of WETH in USDC (borrowPrice) from Sushiswap. Has 8 decimals.
+    /**
+     * @notice Get price of WETH in USDC (borrowPrice) from Sushiswap. Has 6 decimals.
+     * @dev return the price of the borrowed token in asset unit.
+     * @dev returned value will be in same decimals as asset.
+     */
     function _sushiPriceOfBorrow() internal view returns (uint256 borrowPrice) {
         address[] memory path = new address[](2);
         path[0] = address(borrow);
         path[1] = address(asset);
 
-        uint256[] memory amounts = router.getAmountsOut({amountIn: 1e18, path: path});
+        uint256[] memory amounts = router.getAmountsOut({amountIn: 10 ** borrow.decimals(), path: path});
         // We multiply to remove the 0.3% fee assessed in getAmountsOut
-        return amounts[1] * 1000 / 997;
+        return amounts[1].mulDivDown(1000, 997);
```

Comment:
Division already truncates so this isn't necessary. It's fine as a stylistic choice though.